backend/utils/db.py

```diff
@@ -65,10 +65,15 @@ def create_snack(snack: SnackCreateSchema) -> Snack:
     with get_db_connection() as conn:
         cursor = conn.cursor()
         cursor.execute("""
-            INSERT INTO snacks (sku, name, quantity)
-            VALUES (?, ?, ?)
+            INSERT INTO snacks (sku, name, quantity, price, description, category, photo_url)
+            VALUES (?, ?, ?, ?, ?, ?, ?)
             RETURNING *
-        """, (snack.sku, snack.name, snack.quantity if snack.quantity is not None else 1))
+        """, (
+            snack.sku, snack.name, 
+            snack.quantity if snack.quantity is not None else 1, 
+            snack.price, snack.description, 
+            snack.category, snack.photo_url
```

Comment:
just a minor formatting suggestion
```suggestion
            snack.sku, 
            snack.name, 
            snack.quantity if snack.quantity is not None else 1, 
            snack.price, 
            snack.description, 
            snack.category, 
            snack.photo_url